kuka_kr5_support/package.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>kuka_kr5_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA KR5 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for KUKA KR 5 manipulators. This currently includes the KR 5 arc only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>KR 5 arc - Default</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information
+      found in the online datasheet <a href="https://www.kuka.com/-/media/kuka-downloads/imported/48ec812b1b2947898ac2598aff70abc0/spez_kr_5_arc_de.pdf">here</a>. All urdfs are based on the default motion and joint velocity limits,
+      unless noted otherwise.
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+    <p>
+      <b>NB</b>: this package currently uses non-valid inertia parameters.
+    </p>
+  </description>
+  <author>Denis Štogl</author>
+  <maintainer email="denis.stogl@kit.edu">Denis Štogl (KIT, IAR-IPR)</maintainer>
```

Comment:
Could you please also add at least one ROS-I maintainer (me or Shaun)? That way we can be sure that at least one of the maintainers will always be reachable, independent of whether you are still affiliated / work at KIT.